### PR TITLE
9 animations

### PR DESCRIPTION
**Pull request by MaksymSelenkov**
https://maksymselenkov.github.io/frontend-course-2021/homeworks/maksym.selenkov_MaksymSelenkov/9-animations/index.html

### CODE REVIEW

File: homeworks/maksym.selenkov_MaksymSelenkov/9-animations/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const btnEl = document.querySelector('[data-theme]');
+const linkEl = document.querySelector('[data-theme-link]');
+const switchEl = document.querySelector('[data-switch]');
+const btnToTopEl = document.querySelector('[data-to-top]');
+const galleryEl = document.querySelector('[data-gallery]');
+const needsImgEl = document.querySelector('[data-needs-image]');
+const dreamImgEl = document.querySelector('[data-dream-image]');
+
+function changeTheme() {
+    const lightTheme = 'light.css';
```
Comment (DrSmile444) [inline homeworks/maksym.selenkov_MaksymSelenkov/9-animations/index.js:10]: It's better to move such consts to the global scope.

File: homeworks/maksym.selenkov_MaksymSelenkov/9-animations/light.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
+* {
```
Comment (DrSmile444) [inline homeworks/maksym.selenkov_MaksymSelenkov/9-animations/light.css:1]: Apply all comments in this CSS file as well

File: homeworks/maksym.selenkov_MaksymSelenkov/9-animations/style.css [hunk 3]
```diff
@@ -0,0 +1,379 @@
+html {
+  font-family: "Roboto", sans-serif;
+  scroll-behavior: smooth;
+}
+
+a {
```
Comment (DrSmile444) [inline homeworks/maksym.selenkov_MaksymSelenkov/9-animations/style.css:6]: Nooo 😢
You should NEVER use tag selectors in CSS. Create a class and apply it to all elements. Review all your CSS code to replace such tag selectors

**Review by DrSmile444 — CHANGES_REQUESTED** [review]

**Comment by DrSmile444** [comment]
One more thing:
Please, review how your site is responsive. It displays wrong on 1024px, so I didn't check fewer sizes.

maksymselenkov github io_frontend-course-2021_homeworks_maksym selenkov_MaksymSelenkov_9-animations_index html (1)

**Comment by DrSmile444** [comment]
And one more:
You don't have SVG animation on the site load, as is mentioned in the task:

> When the page is loaded, the svg logo must be animated once

**Comment by MaksymSelenkov** [comment]
I have SVG animation in CSS and seems it works or I should add it by window.onload in JS?

**Comment by DrSmile444** [comment]
> I have SVG animation in CSS and seems it works or I should add it by window.onload in JS?

Oh, I fount the animation, my bad, sorry). I just thought it would be like loader, but not a problem.

> As for BEM classes: I write 2 classes when I want to make a block from the element: "page__section section" to use further "section__title", "section__content". In this case I have to remove one of them?

I got the point. So let's keep it as it is.

PS: Still, responsive for iPad looks bad. Could you double-check it, please?

image

**Review by DrSmile444 — CHANGES_REQUESTED** [review]
Almost great!
But you forgot to fix the header. Review the whole page and fix other bugs, if any.

image

**Review by DrSmile444 — APPROVED** [review]
Great job! Thank you!

**Pull request merged**